methods

PenguinPaladin opened the pull request.

PenguinPaladin committed:
modified:   src/Main.java

PenguinPaladin committed:
modified:   src/Main.java

PenguinPaladin committed:
Merge pull request #1 from CoolNickname68/branch2

	new file:   src/homework.java

PenguinPaladin committed:
modified:   src/Main.java

PenguinPaladin committed:
Merge branch 'branch1' of https://github.com/CoolNickname68/Education into branch1

PenguinPaladin committed:
modified:   src/Main.java

PenguinPaladin committed:
modified:   src/Main.java
	deleted:    src/homework.java

pavelilyukevichsky commented on src/Main.java:
```diff
@@ -1,138 +1,71 @@
-                } else {
-                    System.out.println("Место стоячее");
+        if (clientDeviceYear < currentYear) {
+            if (clientDeviceYear < 2015) {
```
2015 это ваша переменная clientDeviceYear в ней будет храниться это значение. оно приходит на вход в ваш метод. поэтому эту проверку удаляем

PenguinPaladin committed:
modified:   src/Main.java

PenguinPaladin committed:
modified:   src/Main.java

PenguinPaladin committed:
modified:   src/Main.java
	new file:   src/ineterface.java
	new file:   src/vehicles.java

PenguinPaladin committed:
new file:   src/Bicycle.java
	new file:   src/Car.java
	modified:   src/Main.java
	new file:   src/ServiceStation.java
	new file:   src/Truck.java
	renamed:    src/ineterface.java -> src/Vehicle.java
	deleted:    src/vehicles.java

PenguinPaladin committed:
modified:   src/Bicycle.java
	modified:   src/Car.java
	modified:   src/Main.java
	modified:   src/ServiceStation.java
	new file:   src/Transport.java
	modified:   src/Truck.java
	modified:   src/Vehicle.java
